### PR TITLE
Added more aknow

### PR DESCRIPTION
**Pull request by lrsantos11**
Regarding #51, please @abelsiqueira @rgaiacs confirm if there is anyone we should thank in addition.

### CODE REVIEW

File: jors_perprof-py.tex [hunk 1]
```diff
@@ -57,8 +57,13 @@ \section{Reuse potential}
 \input{reuse}
 
 \section*{\colorsectionnumberless{Acknowledgemets}}
-We would like to thank the three anonymous reviewers who gave valuable insight
-and suggestions to this paper.
+The authors would like to thank CNPq and FAPESP --- grants 2008/09685-8  and
+2009/XXXXX ---
```
Comment (rgaiacs) [inline jors_perprof-py.tex]: "2009/XXXXX" -> "501763/2013-9"

You can merge after that.

Comment (lrsantos11) [inline jors_perprof-py.tex]: This number is from your FAPESP grant?

Comment (rgaiacs) [inline jors_perprof-py.tex]: This is the number that @abelsiqueira provided.

**Comment by rgaiacs** [comment]
@lrsantos11 you will need to rebase this branch. Thanks in advance.

**Comment by lrsantos11** [comment]
Can you confirm @rgaiacs if it's okay and merge?

File: jors_perprof-py.tex [hunk 2]
```diff
@@ -56,9 +56,15 @@ \section{Availability}
 \section{Reuse potential}
 \input{reuse}
 
-\section*{\colorsectionnumberless{Acknowledgements}}
-We would like to thank the three anonymous reviewers who gave valuable insight
-and suggestions to this paper.
+
+\section*{\colorsectionnumberless{Acknowledgments}}
+The authors would like to thank FAPESP 
+(grants 2008/ 09685-8 and 2009/17273-4) and CNPq (grant 501763/ 2013-9) for the
```
Comment (rgaiacs) [inline jors_perprof-py.tex]: There are extra white spaces at the grant id.

- [ ] `2008/ 09685-8` -> `2008/09685-8`
- [ ] `501763/ 2013-9` -> `501763/2013-9`

File: jors_perprof-py.tex [hunk 3]
```diff
@@ -56,9 +56,15 @@ \section{Availability}
...
+partial support given to this project
+and the people from LPOO and IMECC/Unicamp.
```
Comment (rgaiacs) [inline jors_perprof-py.tex]: Unicamp -> UNICAMP

File: jors_perprof-py.tex (hunk 2 above)
Comment (lrsantos11) [inline jors_perprof-py.tex]: I know, but if it's kept, the final result is outside textwidth. Should we insert a footnote?

**Comment by rgaiacs** [comment]
> Should we insert a footnote?

I move the grant id to footnote and push this change.

**Pull request merged**